### PR TITLE
Improved contrast of price card with white color 

### LINKED ISSUE #4: Improve contrast card price
Improve the contrast of the price card (https://www.lupax.app/pricing), with a white color.

Opera Snapshot_2023-03-29_160710_localhost

### PR DESCRIPTION
**Pull request by TheAayushRana**
fixes #4  The price card color is changed to white color

### CODE REVIEW

**Comment by fermenen** [comment]
@TheAayushRana 
The solution is perfect but the diff have problems, how you have configured the tab key (programmes in studio code)?
You see that the file has many differences..

**Comment by TheAayushRana** [comment]
Actually, it is because I use prettier vs extension which format code itself. Should I again commit with prettier extension closed

**Comment by fermenen** [comment]
@TheAayushRana  ok, yes!

**Comment by TheAayushRana** [comment]
I have push new code Can u check if it is good

File: frontend/pages/pricing.tsx [hunk 1]
```diff
@@ -82,7 +83,7 @@ export default function Pricing() {
                     </Box>
 
                     <VStack
-                        bg={useColorModeValue('gray.50', 'gray.700')}
+                        bg={useColorModeValue('white', 'gray.700')}
```
Comment (fermenen) [inline frontend/pages/pricing.tsx:86]: @TheAayushRana to understand why there are two colours here, the second one is for dark mode (it is not activate in lupax)

**Comment by fermenen** [comment]
@TheAayushRana we can see the change in development env, in https://develop.lupax.app/pricing
Now we must check in this env if all work correctly. (we haven't got integration test)

**Pull request merged**